server/room-battle.ts

```diff
@@ -1211,42 +1173,28 @@ class RoomBattleStream extends BattleStream {
 	}
 }
 
-exports.RoomBattlePlayer = RoomBattlePlayer;
-exports.RoomBattleTimer = RoomBattleTimer;
-exports.RoomBattle = RoomBattle;
-exports.RoomBattleStream = RoomBattleStream;
-
 /*********************************************************
  * Process manager
  *********************************************************/
 
-/** @type {typeof import('../lib/process-manager').StreamProcessManager} */
-const StreamProcessManager = require(/** @type {any} */('../.lib-dist/process-manager')).StreamProcessManager;
-
-const PM = new StreamProcessManager(module, () => {
+export const PM = new StreamProcessManager(module, () => {
 	return new RoomBattleStream();
 });
 
 if (!PM.isParentProcess) {
 	// This is a child process!
-	global.Config = require(/** @type {any} */('../.server-dist/config-loader')).Config;
-	global.Chat = require(/** @type {any} */('../.server-dist/chat'));
+	global.Config = (require('./config-loader') as any).Config;
```

Comment:
i don't think this or below still needs to be `as any`? these have been typescripted?

Reply:
As discussed in https://psim.us/dev, I initially did this to prevent TSLint from complaining about this not using `import`, but it seems like disabling it is the preferred action here.